src/components/WwwFrame/TheHeader.vue

```diff
@@ -810,20 +811,24 @@ export default {
 			hasLentBefore,
 			hasDepositBefore,
 		});
-		window.addEventListener('resize', this.determineIfMobile());
+
+		this.determineIfMobile();
+		window.addEventListener('resize', _throttle(() => {
+			this.determineIfMobile();
+		}, 200));
 
 		if (this.$refs?.newExpHeader) {
 			this.$refs?.newExpHeader?.getSuggestions?.(this.apollo);
 		}
 	},
 	beforeUnmount() {
-		window.removeEventListener('resize', this.determineIfMobile());
+		window.removeEventListener('resize', _throttle(() => {
```

Comment:
Throttle can't be in the remove event listener, since it creates a new instance of the method.